tsconfig.json

```diff
@@ -0,0 +1,19 @@
+{
+  "include": [
+    "src/**/*"
+  ],
+  "exclude": [
+    "src/**/*.test.ts"
+  ],
+  "compilerOptions": {
+    "outDir": "dist/",
+    "target": "es2016",
```

Comment:
question: Why 2016 and not ESNext?

Reply:
Was just the default by tsc init ...

So, it was basically the implicit default before as well, if we just want to support bleeding edge we can set it to ESNext as well of course.